app/utility/transforms/CollapsedMetrics.js

```diff
@@ -364,6 +365,34 @@ export const getSetLossOfDurations = (set) => {
     return getSetLossMetrics(durations);
 };
 
+export const getRPE1rm = (set) => {
```

Comment:
- [ ] what happens when rep length is 0, or rep length is 11+
- [ ] does this account for removed or invalid reps, are you sure
- [ ] does this account for weights that haven't been entered?